src/Converter/Feature2Mesh.js

```diff
@@ -1,31 +1,20 @@
 import * as THREE from 'three';
 import Earcut from 'earcut';
-import Coordinates from 'Core/Geographic/Coordinates';
 import { FEATURE_TYPES } from 'Core/Feature';
+import { deprecatedFeature2MeshOptions } from 'Core/Deprecated/Undeprecator';
 
-function getProperty(name, options, defaultValue, ...args) {
-    const property = options[name];
+const rColor = new THREE.Color();
```

Comment:
It's a small detail but maybe `rColor` is not the most suited name for this constant, since it is not always set to a random color.

Reply:
yes